book/caveats.nim

```diff
@@ -0,0 +1,81 @@
+import nimib, nimibook
+
+
+nbInit
+nbUseNimibook
+
+nbText: """
+# Caveats
+There are some caveats when working with norm that you need to consider and strategies to work around them.
+
+## Managing data for Many-To-Many relationships
+Support for Many-To-Many relationships has not yet been reached. You will have to set-up and manage the necessary "glue"-models yourself as if they were normal models.
```

Comment:
I'm not sure I should get picky with the words really, but “regular” models sounds better to me than “normal.”

Reply:
> I'm mostly happy with the PR. There are a few minor nitpicks but that's about it.
> 
> Also, there's “Caveats” section on the tables tutorial: https://norm.nim.town/tutorial/tables.html
> 
> Maybe it should be moved here.

I noticed that one as well. I'm perfectly happy to move it to the tables-section, though in my own opinion I'm absolutely not sure where I would put it.

The contents of it are caveats for the rows-section more than anything, as they deal for the most part with caveats around  fetching data, so DML. Given that, they seem slightly misplaced in the tables-section, which deals more in DDL.

The rows section meanwhile is already long enough imo, which makes it a tempting proposition to have a general caveats section in itself. Thus, I'm really not sure what would be the best option.

Shall we go with appending it to the tables-section anyway?